src/chrome/content/rules/CNBC.com-mixed.xml

```diff
@@ -1,7 +1,7 @@
 <!--
 	For more CNBC.com related, see CNBC.com.xml
 -->
-<ruleset name="CNBC.com (mixed)">
+<ruleset name="CNBC.com (mixed)" platform="mixedcontent">
```

Comment:
With the mixedcontent platform, you can remove the exclusions and use a simple redirect.
(or you can move the partial redirect to the main ruleset.)